packages/xdl/src/project/ExpSchema.ts

```diff
@@ -1,34 +1,33 @@
-/**
- * @flow
- */
-
 import fs from 'fs';
 import path from 'path';
 import Schemer from '@expo/schemer';
 import Api from '../Api';
-import * as ProjectUtils from './ProjectUtils';
+import * as ConfigUtils from '@expo/config';
+
+export type Schema = any;
```

Comment:
`object` is not very useful here because it doesn't allow accessing any properties. I think it's the correct type only in cases where any object will suffice (e.g. the type of the argument to `Object.keys` builtin method).

I used `any` as a shortcut because the type definitions for packages that `@expo/schemer` uses to validate/traverse JSON Schema didn't include type definitions for the schema JSON. But maybe there's a separate package that we can install to get the full TypeScript type for this format.

Reply:
`object` in TS is different than in Flow (which is the fairly useless type). In TS it is basically `{ [key: string]: any }`.

Reply:
If I change the the type to `object`, I get 4 errors in this file:
```
@expo/xdl: src/project/ExpSchema.ts(31,14): error TS2339: Property 'meta' does not exist on type 'object'.
@expo/xdl: src/project/ExpSchema.ts(31,27): error TS2339: Property 'meta' does not exist on type 'object'.
@expo/xdl: src/project/ExpSchema.ts(34,29): error TS2339: Property 'properties' does not exist on type 'object'.
@expo/xdl: src/project/ExpSchema.ts(78,3): error TS2741: Property 'schema' is missing in type '{}' but required in type '{ schema: object; }'.
```
Maybe you are thinking of some other type or maybe this depends on `tsconfig` settings?